action.yml

```diff
@@ -16,13 +16,16 @@ inputs:
   apiKey:
     required: true
     description: "Replay.io API Key"
+  public:
+    required: true
```

Comment:
It kind of looks to me like we have a (reasonable) default of `false`, so I would expect this *wouldn't* be required? (But maybe I'm misunderstanding what required means here)

Reply:
Good call!